### ISSUE TITLE
faster_rcnn_resnet101_coco: How to get Input and Output nodes from model graph?

### ISSUE THREAD

**Issue by VictorDDT** [tensorflow/models#6852]
### System information
- **What is the top-level directory of the model you are using**:
/opt/develop/repos/object_detection/gray_whales/models/faster_rcnn_resnet101_coco_2018_01_28

- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
No

- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Ubuntu 18.04.2 LTS

- **TensorFlow installed from (source or binary)**:
Tensorflow installed via PIP

- **TensorFlow version (use command below)**:
1.11.0 with GPU support

- **Bazel version (if compiling from source)**:
n/a

- **CUDA/cuDNN version**:
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2017 NVIDIA Corporation
Built on Fri_Sep__1_21:08:03_CDT_2017
Cuda compilation tools, release 9.0, V9.0.176

- **GPU model and memory**:
NVIDIA-SMI 418.43
Driver Version: 418.43
CUDA Version: 10.1
GPU  Name: GeForce RTX 2080
Memory: 7949MiB

- **Exact command to reproduce**:
`
#!/bin/bash

PIPELINE_CONFIG_PATH=/opt/develop/repos/object_detection/gray_whales/models/faster_rcnn_resnet101_coco_2018_01_28/pipeline.config
MODEL_DIR=/opt/develop/repos/object_detection/gray_whales/models/faster_rcnn_resnet101_coco_2018_01_28/
NUM_TRAIN_STEPS=50000
SAMPLE_1_OF_N_EVAL_EXAMPLES=1

python3 /opt/develop/virtualenv-tf/lib/python3.6/site-packages/tensorflow/models/research/object_detection/legacy/train.py \
    --pipeline_config_path=${PIPELINE_CONFIG_PATH} \
    --train_dir=${MODEL_DIR} \
    --num_train_steps=${NUM_TRAIN_STEPS} \
    --sample_1_of_n_eval_examples=$SAMPLE_1_OF_N_EVAL_EXAMPLES \
    --alsologtostderr
`

### Describe the problem

I have trained a model (faster_rcnn_resnet101_coco_2018_01_28) with Tensorflow. I have a ".pb" graph.

To make a frozen graph I need Input and Output nodes.

How can I find it in the graph?

I have got full list of nodes [here](https://raw.githubusercontent.com/VictorDDT/uploadfiles/master/model_nodes.txt).

There no any nodes like 'Softmax', 'Placeholder' as it recommended in other posts.

**Comment by pkulzc** [comment]
Have you tried object_detection/export_inference_graph.py? It should have everything you need.
The file you printed out are graph ops instead of input and output tensors.

**Comment by VictorDDT** [comment]
Thank you it works fine!
```
#!/bin/bash

PIPELINE_CONFIG_PATH=/opt/develop/repos/object_detection/gray_whales/models/faster_rcnn_resnet101_coco_2018_01_28/pipeline.config
MODEL_CHECKPOINT_PATH=/opt/develop/repos/object_detection/gray_whales/models/faster_rcnn_resnet101_coco_2018_01_28/model.ckpt-200000
MODEL_DIR=/opt/develop/repos/object_detection/gray_whales/models/faster_rcnn_resnet101_coco_2018_01_28/

python3 /opt/develop/virtualenv-tf/lib/python3.6/site-packages/tensorflow/models/research/object_detection/export_inference_graph.py \
    --input_type=image_tensor \
    --pipeline_config_path=${PIPELINE_CONFIG_PATH} \
    --trained_checkpoint_prefix=${MODEL_CHECKPOINT_PATH} \
    --output_directory=${MODEL_DIR} \
```
